mapie/calibrators/ccp/base.py

```diff
@@ -40,10 +41,12 @@ class CCPCalibrator(BaseCalibrator, metaclass=ABCMeta):
         List of functions (or ``CCPCalibrator`` objects) or single function.
 
         Each function can take a combinaison of the following arguments:
+
         - ``X``: Input dataset, of shape (n_samples, ``n_in``)
         - ``y_pred``: estimator prediction, of shape (n_samples,)
         - ``z``: exogenous variable, of shape (n_samples, n_features).
-            It should be given in the ``fit`` and ``predict`` methods.
+          It should be given in the ``fit`` and ``predict`` methods.
```

Comment:
Do we perform a test for this? By that, I mean that we provide the same combination for the fit and predict.

Reply:
I am not sure to understand correctly, but if the calibrator need a ``z`` value, it will not work neither in the ``fit`` nor ``predict`` if it is not given. So in a way, there can't be an issue of forgotten ``z`` only in the ``fit`` or ``predict``